paymentsheet/src/main/java/com/stripe/android/paymentsheet/forms/PaymentMethodRequirements.kt

```diff
@@ -266,6 +266,6 @@ internal val AffirmRequirement = PaymentMethodRequirements(
  */
 internal val AuBecsDebitRequirement = PaymentMethodRequirements(
     piRequirements = emptySet(),
-    siRequirements = emptySet(),
+    siRequirements = null,
```

Comment:
What is the difference between null and emptySet here?

Reply:
null means that it is not supported, emptySet would mean it is supported but has no additional requirements.